wp/wp-content/themes/lahma_maker/Showsschedule.php

```diff
@@ -57,7 +57,6 @@
         <li>12:00–14:00 <strong><a href="../shows/mukodo-mukod/">Működő Működ</a></strong></li>
         <li>14:00-14:30 <strong><a href="../shows/radio-melania/">Rádió Melánia</a></strong></li>
         <li>14:30-16:00 <strong><a href="../shows/mindentbele-szusivacsi/">Mindentbele Szusivacsi</a></strong></li>
-        <li>16:00–17:00 <strong><a href="../shows/recently-played-music/">Recently Played Music</a></strong></li>
```

Comment:
@gammaw – didn't we think about doing all the rare shows with commented out solution? that we can easily switch between shows on the fly from the server? By all means, we can put it also to .gitignore for example

Reply:
could be some .sh script also on the server —> we log in and run it, eg. `thursday-recently.sh` which replaces the comment to the appropriate line? or we don't bother now before the rework release?

Reply:
Yeah, I wouldn't bother with implementing automation for the current website, as it most probably be an arcsi based solution for the next website (see our respective discussion elsewhere). 

Either way, this PR is independent of new ways of automation, it follows the rules that we accepted as the current solution, which is to manually replace the lines in the schedule when a new episode is submitted for the rare show slot (in this case Fri 12am-2pm).